pom.xml

```diff
@@ -912,7 +912,7 @@
 					<!-- Plugin to assemble a jar with dependencies -->
 					<plugin>
 						<artifactId>maven-assembly-plugin</artifactId>
-						<version>2.6</version>
+						<version>3.0.0</version>
 							<configuration>
```

Comment:
@valib A new major version might bring big changes. Have you checked what the impact is?